crates/python/src/lib.rs

```diff
@@ -0,0 +1,30 @@
+use gq_core::data::DataType;
+use gq_core::query::Query;
+use pyo3::exceptions::PyValueError;
+use pyo3::prelude::*;
+
+/// Formats the sum of two numbers as string.
+#[pyfunction]
+fn gq(query: &str, data: &str) -> PyResult<String> {
```

Comment:
TODO: this creates bindings with kwargs. Is this desirable? or should we use positional args